models/petition.py

```diff
@@ -9,105 +9,112 @@
 from datetime import date, timedelta
 
 class Petition(db.Model):
+    
     user = db.ReferenceProperty(User, required=True)
+    name = db.StringProperty(required=True)
     date_added = db.DateProperty(auto_now=True)
-    title = db.StringProperty(required=True)
-    note = db.TextProperty()
-    organizer_email = db.StringProperty(required=True)
-    organizer_name = db.StringProperty(required=True)
-    votes = db.IntegerProperty(default = 0)
-    voters = db.ListProperty(str)
-
+    email = db.StringProperty(required=True)
+    message = db.TextProperty()
+    election = db.StringProperty(required=True)
+    organization = db.StringProperty(required=True)
+    position = db.StringProperty(required=True)
+    signature_num = db.IntegerProperty(default = 0)
+    signatures = db.ListProperty(str)
+    
     def to_json(self):
         return {
             'user': self.user,
+            'name': self.name,
             'id': str(self.key()),
-            'title': self.title,
-            'note': self.note,
-            'votes': self.votes,
-            'organizer_email': self.organizer_email,
-            'organizer_name': self.organizer_name
-        }
-    def get_votes(self):
-        return self.votes
-
-    def get_voters(self):
-        return self.votes
-
+            'email': self.email,
+            'message': self.message,
+            'election': self.election,
+            'organization': self.organization,
+            'position': self.position,
+            'signature_num': self.signature_num,
+            'signatures': self.signatures
+           }
+
+    def get_signature_num(self):
+        return self.signature_num
+    def get_signatures(self):
+        return self.signatures
     def get_user(self):
         return self.user
-
-    def get_org_email(self):
-        return self.organizer_email
-
-    def get_org_name(self):
-        return self.organizer_name
-
-    def add_voter(self,voter):
-        net_id = voter.get_id()
-        if net_id not in self.voters:
-            self.voters.append(net_id)
-            self.votes = self.votes + 1
-            self.put()
-            return True
+    def get_election(self):
+        return self.election
+    def get_organization(self):
+        return self.organization
+    def get_position(self):
```

Comment:
be consistent w/ spacing between functions. 1 space between class methods, 2 between global functions